exportdbs

```diff
@@ -19,9 +19,10 @@ function exportdbs() {
         dbstuff="$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 7 | head -n 1)"
         name="${cpuser}_${dbstuff}"
         uapi Mysql create_user name=$name password=$pass 1>/dev/null
+        host="$(uapi Mysql locate_server remote_host 2>/dev/null | while read x; do echo $x | grep remote_host | cut -f1 | sed 's@remote_host: @@'; done;)"
         for x in $(echo $dbs); do
             uapi Mysql set_privileges_on_database user=$name database=$x privileges=ALL%20PRIVILEGES 1>/dev/null
-            mysqldump -u $name -p$pass $x >~/migrationdbs/$x.sql
+            mysqldump -u $name -p$pass -h=$host $x >~/migrationdbs/$x.sql
```

Comment:
if we are changing the variable name from $host to $mysql_host, it will need to be updated in all instances like this.